### PR TITLE
toggle password visibility

### LINKED ISSUE #1354: Toggle password visibility
On log in section add a button that hide and show the password. Its helpful for the users to know what they are writing. Here is an example: 

maxresdefault

### PR DESCRIPTION
**Pull request by nmro184**
resolving #1354

### CODE REVIEW

File: src/components/Account/Login/Login.component.js [hunk 1]
```diff
@@ -32,7 +35,8 @@ export class Login extends Component {
 
   state = {
     isLogging: false,
-    loginStatus: {}
+    loginStatus: {},
+    showPassword: false
```
Comment (shayc) [inline src/components/Account/Login/Login.component.js]: nit: `showPassword` sounds like a fn name (an action), would you mind renaming to something more boolean in nature? perhaps `isPasswordVisible`

Comment (nmro184) [reply src/components/Account/Login/Login.component.js]: yes I agree

File: src/components/UI/FormItems/TextField.js [hunk 2]
```diff
@@ -3,24 +3,36 @@ import PropTypes from 'prop-types';
...
 
 const defaultProps = {
   className: '',
-  error: false
+  error: false,
+  endAdornment: null
 };
 
-const TextField = ({ className, error, ...props }) => (
+const TextField = ({ className, error, endAdornment, ...props }) => (
```
Comment (RodriSanchez1) [inline src/components/UI/FormItems/TextField.js]: You are sending the `InputProps` prop that contains the `endAdornment`. So the correct prop to include here its the `InputProps`

File: src/components/UI/FormItems/TextField.js [hunk 3]
```diff
@@ -3,24 +3,36 @@ import PropTypes from 'prop-types';
...
-  ])
+  ]),
+  endAdornment: PropTypes.node
 };
 
 const defaultProps = {
   className: '',
-  error: false
+  error: false,
+  endAdornment: null
```
Comment (RodriSanchez1) [inline src/components/UI/FormItems/TextField.js]: Change this to `InputProps: null`. See  line 24 comment. Also update the `propTypes` above

File: src/components/UI/FormItems/TextField.js [hunk 4]
```diff
@@ -3,24 +3,36 @@ import PropTypes from 'prop-types';
...
 };
 
-const TextField = ({ className, error, ...props }) => (
+const TextField = ({ className, error, endAdornment, ...props }) => (
   <FormControl className={className} error={!!error}>
-    <MUITextField error={!!error} {...props} />
+    <MUITextField
+      error={!!error}
+      {...props}
+      InputProps={{
```
Comment (RodriSanchez1) [inline src/components/UI/FormItems/TextField.js]: If you make the changes that i suggested use  `InputProps={InputProps}` instead.

**Review by RodriSanchez1 — CHANGES_REQUESTED** [review]
I get this error on the console `TextField.js:43 Warning: Failed prop type: The prop children is marked as required in ForwardRef(InputAdornment), but its value is null.`

Let me know if you have some doubts!!

**Comment by nmro184** [comment]
thanks for the comments , when I run it locally I didn't get that warning , committed the changes you suggested.

**Comment by nmro184** [comment]
@RodriSanchez1 hey I see the changes requested tag is still present. Is there anything else that needs to be fixed?

**Review by RodriSanchez1 — APPROVED** [review]
Good work!!! Thanks!

**Pull request merged**